packages/vite/src/node/config.ts

```diff
@@ -547,6 +547,7 @@ export interface InlineConfig extends UserConfig {
   configFile?: string | false
   /** @experimental */
   configLoader?: 'bundle' | 'runner' | 'native'
+  /** @deprecated */
   envFile?: false
```

Comment:
I think we need to set `envDir: false` when `envFile: false` is set for backward compat.